docs/codacy-api/api-tokens.md

```diff
@@ -4,23 +4,24 @@ description: Create an account API token to authorize access to all the reposito
 
 # API tokens
 
-The Codacy API tokens allow you to:
+Codacy provides **account** and **project**-level API tokens that allow you to:
 
+-   [Upload coverage data](../coverage-reporter/index.md) to Codacy
+-   [Upload the results of running local analysis](../related-tools/local-analysis/running-local-analysis.md) to Codacy
 -   [Authenticate when using the Codacy API](using-the-codacy-api.md#authenticating-requests)
 
--   [Upload coverage data](../coverage-reporter/index.md) or [upload the results of running local analysis](../related-tools/local-analysis/running-local-analysis.md) to Codacy
+The sections below provide details about the two types of API tokens and instructions on how to generate and revoke them.
 
-Codacy provides two types of API tokens:
+{% include-markdown "../assets/includes/api-token-warning.md" %}
 
--   **Account API tokens** are defined at the Codacy user account level. Each account API token authorizes access to the same organizations, repositories, and operations as the [roles and permissions of the owner of the account](../organizations/roles-and-permissions-for-synced-organizations.md).
+## Generating and revoking account API tokens {: id="account-api-tokens"}
 
--   **Project API tokens** are defined on individual repositories. Each project API token only authorizes access to the corresponding repository.
+Account API tokens are defined at the **Codacy user account level**. Each account API token authorizes access to the same organizations, repositories, and operations as the [roles and permissions of the owner of the account](../organizations/roles-and-permissions-for-synced-organizations.md).
```

Comment:
Before, the explanation of how account API tokens worked was in the intro. This meant that when users arrived at this page directly on the account API tokens section, they wouldn't see this explanation.

I'd like to test if this change is helpful to clarify the implications of using account API tokens during the setup of coverage.